Library/Homebrew/formula_installer.rb

```diff
@@ -777,13 +777,18 @@ def pour
     end
 
     keg = Keg.new(formula.prefix)
+    tab_file = formula.prefix.join(Tab::FILENAME)
+    # Skip the cache since the receipt will be rewritten
```

Comment:
I think it would be clearer to nuke the cache before rereading rather than use a method that's not using the cache (and should probably be `private` as a result).